help/de/Help_ch_Customize.xml

```diff
@@ -2814,48 +3664,97 @@
+
+          <imageobject role="fo">
+            <imagedata fileref="figures/Help_Reset_Warnings.png" 
+                         srccredit="Christian Wehling"></imagedata>
+          </imageobject>
+        </mediaobject>
+      </screenshot>
+    </figure>
+
+    <para>Warnungen können selektiv aktiviert werden wobei der Dialog selbsterklärend in zwei Rubriken
+      aufgeteilt ist.
+    </para>
+
+    <para><variablelist spacing="compact">
+        <varlistentry>
+          <term><guilabel>Dauerhafte Warnung</guilabel></term>
+
+          <listitem>
+            <para>Hier werden die Warnungen aufgelistet, die Sie dauerhaft nicht wieder sehen wollten.
+            </para>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term><guilabel>Zeitweise Warnung</guilabel></term>
+
+          <listitem>
+            <para>Unter den zweitweisen Warnung fallen die Hinweise, die Sie lediglich für die aktuelle Sitzung nicht
+              mehr sehen wollen.
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
     </para>
   </sect1>
 
   <sect1  id="chang-lang">
-    <title>Changing the Language or Region</title>
+    <title>Ändern der Sprache oder des Gebietsschemas</title>
 
-    <para>The language of the &app; user interface is not modifiable directly from the program&rsquo;s
-      preferences.
+    <para>Die Sprache der &app; Benutzeroberfläche kann nicht direkt über die Programmeinstellungen
+      geändert werden.
     </para>
 
-    <para>The way you can change the language depends on the operating system you are running &app; on.
+    <para>Die Art und Weise, wie Sie die Sprache ändern können, hängt von dem Betriebssystem ab, auf dem
+      Sie &app; ausführen.
     </para>
 
-    <variablelist>
+    <variablelist spacing="compact">
       <varlistentry>
         <term>&lin;</term>
 
         <listitem>
-          <para>In general you should set the <envar>LANGUAGE</envar> and <envar>LANG</envar> environment variables
-            before starting &app;. To do this you need to open a terminal and run the following
-            command:
+          <para>Im Allgemeinen sollten Sie die Umgebungsvariablen <envar>LANGUAGE</envar> und <envar>LANG</envar>
+            setzen, bevor Sie &app; starten. Dazu müssen Sie ein Terminal öffnen und den folgenden
+            Befehl eingeben:
           </para>
 
           <para><command>LANGUAGE=<replaceable>ll_LL</replaceable> LANG=<replaceable>ll_LL</replaceable>
             gnucash</command>
           </para>
 
-          <para><replaceable>ll_LL</replaceable> is the locale you want to run &app; with
-            (<emphasis>de_DE</emphasis> for Deutsch, <emphasis>it_IT</emphasis> for Italian etc.)
+          <para><replaceable>ll_LL</replaceable> ist das Gebietsschema, in dem Sie &app; ausführen möchten; mit
+            <emphasis>de_DE</emphasis> für Deutsch, <emphasis>it_IT</emphasis> für Italienisch
+            usw.
```

Comment:
gefährliche, alte Beispiele! Die häufigsten Verwechsler:
be_BE: weißrussisch in Belgien
uk_UK: ukrainisch im Vereinigten Königreich

Echte Beispiele:
Ostbelgien: de_BE
Südtirol: de_IT

Schweiz: {de|fr|it|rm}_CH, wobei rm in Gnucash nicht verfügbar

pt_BR brasilianisches Portugisisch
pt: echtes Portugisisch von Portugal bis Ost-Timor